script/GmOracleWithAggregator.s.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "utils/BaseScript.sol";
+import "oracles/GmOracleWithAggregator.sol";
+
+contract GmOracleWithAggregatorScript is BaseScript {
+
+    function deploy() public returns (GmOracleWithAggregator oracle) {
+        if (block.chainid == ChainId.Arbitrum) {
+            vm.startBroadcast();
+
+            oracle = new GmOracleWithAggregator(
```

Comment:
if you user deployer.deploy_<contract name> you can get the deployment file as well